src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private int side;
+    private int base;
+    private String color = getColor();
```

Comment:
You should avoid initializing instance variables with method calls like `getColor()` directly. This should be set in the constructor or a setter method to maintain clarity and control over the class's properties.